spec/brcobranca/boleto/sicredi_spec.rb

```diff
@@ -145,4 +144,23 @@
       expect(File.exist?(tmp_file.path)).to be_falsey
     end
   end
+
+  it 'quando dígito verificador for 10 deve ser mapeado para 0', focus: true do
+    attributes = @valid_attributes.merge({
+      convenio: '2442725',
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.